examples/auth-with-stytch/next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  reactStrictMode: true,
+  env: {
```

Comment:
Can we use a `.env` file here instead?

Reply:
cd855acc612f811a4d311cfc31d67ada71632c00